integration_tests/test_str_attributes.py

```diff
@@ -25,6 +25,26 @@ def strip():
     assert "     AASAsaSas    " .lstrip() == "AASAsaSas    "
     assert "     AASAsaSas    " .strip() == "AASAsaSas"
 
+def find():
+    s: str
+    sub: str
+    s = "AaaaAABBbbbbBB!@12223BN"
+    sub = "@"
+    assert s.find(sub) == 15
+    assert s.find('B') == 6
+    assert "empty strings" .find("string") == 6
+    s2: str
+    s2 = "Ferrars all spirits his imagine effects amongst neither. It bachelor cheerful of mistaken. Tore has sons put upon wife use bred seen. Its dissimilar invitation ten has discretion unreserved. Had you him humoured jointure ask expenses learning. Blush on in jokes sense do do. Brother hundred he assured reached on up no. On am nearer missed lovers. To it mother extent temper figure better."
```

Comment:
What's the source of this string?

Reply:
Coming from https://www.rs-niers-mg.de/breaking-bad-illustrated/?

Well copying a string from a website makes us prone to copyright claims. Can you just write something of your own? Like just take this review comment and put it as a string?

Reply:
it's from a text generator, but i will change it.

Reply:
@czgdp1807 
ready now.